Project.toml

```diff
@@ -1,6 +1,6 @@
 name = "Zygote"
 uuid = "e88e6eb3-aa80-5325-afca-941959d7151f"
-version = "0.5.12"
+version = "0.5.10"
```

Comment:
Our last actual release was 0.5.9,
looks like we got a bit a head of ourselves.
Once this is merged i will tag a release

Reply:
This habit of changing the version number in each PR doesn't scale very well, creates too many conflicts. 
What about a separate PR (or commit straight to master) with the version bump and then the registrator tagging?
It's an easy process, can be done in the browser

Reply:
Another solution might be to just always register the new version right away after such a PR is merged.